### ISSUE TITLE
Không vào được album

### ISSUE THREAD

**Issue by motcaiten** [dangdinhtu2014/nukeviet_photo#8]
Em cài trền bản nk 4.0.11 thì thấy khi click nó ko vào được album và đường link nó có dạng (Nó bị 2 dấu "/")

http://domain.com/photo//khu-luu-tru-cong-nhan-1.html

**Comment by dangdinhtu2014** [comment]
Bạn kiểm tra xem đã tạo chuyên mục cho nó chưa ?

**Comment by motcaiten** [comment]
À cái này do lỗi block nó không nhận.

Với một điều nữa là nếu không bật block lên thì khi vào link http://domain.com/photo/ thì trắng không có gì

**Comment by dangdinhtu2014** [comment]
Bạn cấu hình trong admin theo dạng chuyên mục nhé. 2 phương án hiển thị kia mình chưa làm

**Comment by motcaiten** [comment]
Vâng. Hiện tại đã được. Anh cho em hỏi luôn là làm sao khi vào xem từng album thì đưa thông tin mô tả album hiện lên. Và khi xem từng hình ảnh thì có dòng miêu tả hình ảnh hiện lên.

Cảm ơn anh.

**Comment by dangdinhtu2014** [comment]
Mẫy bữa nay mình hơi bận để vài bữa nữa mình rảnh xem lại module này rồi trả lời bạn
